packages/core/src/utils/spanUtils.ts

```diff
@@ -1,6 +1,10 @@
 import type { Span, SpanTimeInput, TraceContext } from '@sentry/types';
 import { dropUndefinedKeys, generateSentryTraceHeader, timestampInSeconds } from '@sentry/utils';
 
+export const TraceFlagNone = 0x0;
+// eslint-disable-next-line no-bitwise
+export const TraceFlagSampled = 0x1 << 0;
```

Comment:
l: Should these be all uppercase?

Reply:
good point, makes more sense!

Reply:
also, `0x1 << 0` is just `0x1`, no?

Reply:
Yeah, I think so, I copied this from OTEL, not sure why they have this weird syntax - it is the same, right? 🤔

Reply:
I think so. Maybe they do it just for clarity? 
```js
0x1 << 0 // sampled flag
0x1 << 1 // another flag
```
but yes, it's identical

```sh
> 0x1 === 0x1 << 0
true
```

Reply:
I updated it and added a comment that this is aligned to how this is in OTEL!